cmd/snap-confine/snap-confine.c

```diff
@@ -545,6 +562,7 @@ int main(int argc, char **argv)
 	}
 	// Restore process state that was recorded earlier.
 	sc_restore_process_state(&proc_state);
+	sc_explain("Executing %s", invocation.executable);
```

Comment:
This should probably include some stop marker for the output produced by `explain`.

Reply:
> mborzecki: thanks for looking at --explain,
> mborzecki: the marker, did you mean to use some kind prefix for each line
> mborzecki: or something else?

Reply:
As discussed on IRC, I'd try adding something like
```
<< snap explain start >>
...
<< snap explain end >>
... application output follows
```
IIIRC the kernel uses markers like:

```
-------------------- [ cut here ] --------------------
<backtrace>
-------------------- [ cut here ] --------------------
```
The upside is that you don't have to prefix every line, and it's distinct from the application.